File: c++/program3.c
#include<stdio.h>
#include<stdbool.h>

bool EvenOdd(int nValue)
{

if((nValue%2)==0)
{
    return true;
}
else
{
    return false;
}

}

int main()
{
    int No1=0;
    bool Ans=false;

    printf("Enter Number=");
    scanf("%d",&No1);

    Ans=EvenOdd(No1);
   if(Ans==true)
   {
    printf("%d is an number Even number");
   }
   else
   {
    printf("%d is number an odd number");
   }
   return 0;
}

File: java/program3.java
import java.util.*;
import java.util.Scanner;
class main
{
    public static void main(String arg[])
    {
       Scanner sc=new Scanner (System.in);
       System.out.println("Enter your name=");
       int No1=sc.nextInt();

      if  (No1%2==0)
      {
        System.out.println("This value is even"+No1);
      }
       else
     {
         System.out.println("This value is odd "+No1);
     }
    }
}